mdpow/tests/test_dihedral.py

```diff
@@ -93,3 +93,8 @@ def test_ValueError_different_ensemble(self):
                            match='Dihedral selections from different Ensembles, '):
             DihedralAnalysis([dh1, dh2])
 
+    def test_single_universe(self):
+        dh = self.Ens.select_atoms('name C4', 'name C17', 'name S2', 'name N3')
+        with pytest.raises(NotImplementedError):
+            DihedralAnalysis([dh])._single_universe()
+
```

Comment:
This test is not strictly necessary but we can leave it in, as a check that the DihedralAnalysis does not do something weird to `_single_universe()`.